provider/cmd/pulumi-resource-vault/generate.go

```diff
@@ -49,9 +50,13 @@ func main() {
 		log.Fatalf("cannot reserialize schema: %v", err)
 	}
 
-	err = ioutil.WriteFile("./schema.go", []byte(fmt.Sprintf(`package main
-var pulumiSchema = %#v
-`, versionedContents)), 0600)
+	// Clean up schema.go as it may be present & gitignored and tolerate an error if the file isn't present.
+	err = os.Remove("./schema.go")
+	if err != nil && !errors.Is(err, fs.ErrNotExist) {
+		log.Fatal(err)
+	}
```

Comment:
Why do we need this? It should never be in the CI machines right?

Reply:
Being 100% honest here... I copied this directly from here https://github.com/pulumi/pulumi-aws/commit/b7dd62be2cbc5c437e8e0ffcb45c8bddcb3835b1

Reply:
I think it should be safe to remove this IMO.